tests/examples/default.nix

```diff
@@ -46,7 +46,9 @@
           && [ "$(nix flake show --json | jq 'select(.packages."x86_64-linux".default.name)')" != "" ]; then
         nix eval --read-only --no-allow-import-from-derivation .#default.name
       fi
-      nix flake check "$@"
+      if [ "$(nix flake show --json | jq 'select(.packages."x86_64-linux".resolveImpure)')" == "" ]; then
```

Comment:
`resolveImpure` will not vanish, even if impurities have been resolved. We want to give the user the possibility to re-resolve, as this can be used to update the dependencies.

I think this change should not be necessary. If the tests were failing, then there is probably a problem caused by `src/default.nix` which should be fixed